src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -505,17 +505,24 @@ class TcgStatics {
     }
     return null;
   }
-  static evolve (PokemonCardSet pcs, Card card, ActivationReason reason=ActivationReason.PLAY_FROM_HAND) {
+  static evolve (PokemonCardSet pcs, Card card, ActivationReason reason=PLAY_FROM_HAND) {
     bg().em().run(new Evolve(pcs, card));
     bg().em().run(new CantEvolve(pcs, bg().getTurnCount()));
     bg().em().run(new ActivateAbilities((PokemonCard) card, pcs, reason));
   }
-  static devolve (PokemonCardSet pcs, Card card){
+  static devolve (PokemonCardSet pcs, Card card, CardList newLocation){
     bg().em().run(new Devolve(pcs));
+    bg().em().run(new MoveCard(card, newLocation));
     if (all.contains(pcs)) { //not dead yet.
+      bc "$card Devolved"
       bg().em().run(new RemoveFromPlay(pcs, new CardList(card)));
       bg().em().run(new CantEvolve(pcs, bg().getTurnCount()));
-//			bg().em().run(new ActivateAbilities(pcs.getTopPokemonCard(), pcs, ActivationReason.OTHER));
+
+      // Remove the highest stage non-level up card if devolved from level-up
+      if (card.cardTypes.is(LEVEL_UP) && pcs.cards.filterByType(POKEMON).size() > 1) {
```

Comment:
Not sure of where it'd need to go, but a special check for Actual Pokémon cards should be implemented for stuff like this and `isEvolution()` from PCS.java, as to cover cards like "Ditto (Boundaries Crossed)" (I feel there was some other card like this, but doesn't come to mind right now)